SR2021/2021-01-19-league-1-absent.md

```diff
@@ -0,0 +1,29 @@
+---
+to: Teams who didn't attend session 1
+subject: Student Robotics League 1
+---
+
+Hello there!
+
+We noticed at least one of your teams was absent from our first league session last weekend. We wanted to reach out to see if there was any more support we could provide you, or questions we could answer?
+
+## Assistance
+
+If you're struggling with your robot, here's a few resources which might help you out:
+
+- [The Docs](https://studentrobotics.org/docs/)
+- [Discord](https://studentrobotics.org/docs/team_admin/discord), to get help from other teams / blue shirts
```

Comment:
Does "blue shirts" need explaining here? Perhaps "volunteers" is simpler?
```suggestion
- [Discord](https://studentrobotics.org/docs/team_admin/discord), to get help from other teams / volunteers
```

Reply:
We very frequently use the term blue shirts to reference volunteers, I don't think it needs explaining.